test/scraping/cycling/procyclingstats/races.test.js

```
@@ -0,0 +1,33 @@
+  {
+    filterYear: 2025,
+    filterClass: "2.UWT",
+    input: "test/scraping/cycling/procyclingstats/html/races-2025-2.UWT.html",
+    output:
+      "test/scraping/cycling/procyclingstats/fixtures/races-2025-2.UWT.json",
+  },
+])(`$filterYear $filterClass races`, (data) => {
+  let html, expectedResults;
+
+  beforeAll(async () => {
+    const input = Bun.file(data.input);
```

ESLint

[error] 'Bun' is not defined.

(no-undef)